Plugin.php

```diff
@@ -184,6 +185,16 @@ public function registerFormWidgets()
         ];
     }
 
+    public function localeUrl($url, $locale)
+    {
+        $translator = Translator::instance();
+        $url_parts = parse_url($url);
+        $path = array_get($url_parts, 'path');
+        $parts = array();
+        $parts['path'] = '/' . $translator->getPathInLocale($path, $locale);
+        return http_build_url($url_parts, $parts);
```

Comment:
@mjauvin Just more a code cleanup, but lines 193-195 could be simplified to:
```suggestion
        return http_build_url($url_parts, [
            'path' => '/' . $translator->getPathInLocale($path, $locale)
        ]);
```